pkg/controllers/namespace/namespace_controller.go

```diff
@@ -82,6 +84,54 @@ func (r *NamespaceReconciler) createVPCCR(ctx *context.Context, obj client.Objec
 	return vpcCR, nil
 }
 
+func (r *NamespaceReconciler) createDefaultSubnetSet(ns string) error {
```

Comment:
this seems under each NS, including the NS which use shared VPC, also create Default SubnetSet？

Reply:
done.